android-components/components/ui/autocomplete/src/main/java/mozilla/components/ui/autocomplete/InlineAutocompleteEditText.kt

```diff
@@ -849,9 +855,15 @@ open class InlineAutocompleteEditText @JvmOverloads constructor(
         }
     }
 
+    private fun isNightMode(): Boolean {
+        val darkFlag = resources?.configuration?.uiMode?.and(Configuration.UI_MODE_NIGHT_MASK)
+        return darkFlag == Configuration.UI_MODE_NIGHT_YES
+    }
+
     companion object {
         internal val AUTOCOMPLETE_SPAN = NoCopySpan.Concrete()
-        internal const val DEFAULT_AUTOCOMPLETE_BACKGROUND_COLOR = 0xffb5007f.toInt()
+        internal const val DEFAULT_AUTOCOMPLETE_BACKGROUND_COLOR_DARK = 0xffb5007f.toInt() // dark theme color
+        internal const val DEFAULT_AUTOCOMPLETE_BACKGROUND_COLOR_LIGHT = 0xffBAA8E6.toInt() // light theme color
```

Comment:
I think we can avoid doing anything in code and use two new color resources, one for the default configuration, one for the `night` one.
A-C resources use a naming scheme starting with `mozac` so in this case looks like we only need one new color, maybe `mozac_toolbar_autocomplete_background` which can have the two default values in A-C and then two default values in Focus.
I understand that for Fenix the A-C defaults are okay?

Reply:
Later saw that these are used in an attribute defined by us.
So looks like we can use that attribute as mentioned in https://github.com/JohanLorenzo/android-components/commit/6eae50522fded1985b4c9c8aacb1c7193d618216